src/index.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react"
 import ReactDOM from "react-dom"
-import { App } from "./components/App"
+import { App, callAPI } from "./components/App"
 import "./index.css"
 import * as serviceWorker from "./serviceWorker"
 
-ReactDOM.render(<App />, document.getElementById("root"))
+ReactDOM.render(<App makeRequest={callAPI} />, document.getElementById("root"))
```

Comment:
For funzies—you could add this as a default prop so that you only have to pass it when in a test?

Reply:
I'll look into that 👀